experimental/delegate/README.md

```diff
@@ -0,0 +1,39 @@
+# AIE Delegates
+
+## Introduction
+
+As an early demonstration of the ability to combine executable code for
+heterogenious devices, here we partition a simple model such that most of the
+model compiles for running on CPU, except for the model's matmul, which uses
+a pre-built AIE kernel.
+
+## Running the Demo
+
+### Compile the model
+
+First, log in to a Ryzen AI machine (aka IPU, Phoenix).
+
+```
+# Set up the shell
+cd <workspace root (containing iree-build)>
+source /opt/xilinx/xrt/setup.sh
+export PATH=$PATH:$PWD/iree-build/tools:$PWD/llvm-build/bin
+
+# Compile the model
+cd iree-amd-aie/experimental/delegate
+iree-compile --iree-preprocessing-transform-spec-filename=mlp_spec.mlir mlp.mlir -o mlp.vmfb
+```
+
+### Run the model
+
+```
+# Circumvent xclbin security
+export XRT_HACK_UNSECURE_LOADING_XCLBIN=1
+
+iree-run-module --device=local-sync \
+  --executable_plugin=../../../iree-build/runtime/plugins/AMD-AIE-experimental/delegate/mlp_bf16_aie_delegate.so \
```

Comment:
nit: say {$PATH_TO_IREE_BUILD}/runtime....

Reply:
Fixed, but not exactly in the way requested.  I've kept the default path elsewhere in the instructions, as we have plans to turn our internal e2e IREE/AIE build doc into a public doc.  It sets up a specific directory structure for the sake of simplifying the build process for users.